user/enterprise/prerequisites.md

```diff
@@ -25,6 +25,12 @@ RAM and 8 CPUs**.
 
 If you're running on EC2, we recommend the **c4.2xlarge** instance type for both **Platform** and **Worker**. We also recommend using an image that uses EBS for the root volume, as well as allocating 40 gigs of space to it. It is also recommended _not_ to destroy the volume on instance termination.
 
+For [high availability [HA](/user/enterprise/high-availability/) configurations, you will also need to
+provide your own [Redis](https://redis.io/), [RabbitMQ](https://www.rabbitmq.com/), 
+and [Postgres](https://www.postgresql.org/) instances. We've had some good luck with 
```

Comment:
> We've had some good luck with 
+[compose.com](https://compose.com/) if you would like these services hosted outside 
+your organization.

luck in what? As in we've managed to do this on compose before? :-p

Reply:
We have used compose.com for this and I know of at least one customer who does too.

But, "luck" might be a bit informal, I'll try something else